client/js/preload/goldstoneRouter.js

```diff
@@ -37,6 +37,7 @@ var LauncherView = Backbone.View.extend({
 var GoldstoneRouter = Backbone.Router.extend({
     routes: {
         "api_perf/report": "apiPerfReport",
+        "apps/opentrail": "openTrail",
```

Comment:
A couple things here:
a) sync the path with whatever we finalize as the API name (addons seems to be the most likely).
b) would like to see this be dynamic somehow.  Seems like the router needs uncoded knowledge of the addon to add it to the route.  This will not work if 3rd parties are writing addons.  Do we have a strategy for reading the value from either the data reported from the API or a JavaScript file specific to the addon?